x-pack/plugins/apm/server/routes/errors/route.ts

```diff
@@ -18,9 +18,12 @@ import {
   comparisonRangeRt,
 } from '../default_api_types';
 import { createApmServerRouteRepository } from '../apm_routes/create_apm_server_route_repository';
+import { getServiceErrorGroupMainStatistics } from './get_error_groups/get_error_group_main_statistics';
+import { getServiceErrorGroupPeriods } from './get_error_groups/get_error_group_detailed_statistics';
+import { getErrorGroupSample } from './get_error_groups/get_error_group_sample';
 
-const errorsRoute = createApmServerRoute({
-  endpoint: 'GET /internal/apm/services/{serviceName}/errors',
+const errorsMainStatisticsRoute = createApmServerRoute({
+  endpoint: 'GET /internal/apm/services/{serviceName}/errors/main_statistics',
```

Comment:
I think we should stay consistent with the transaction groups api's which are:
```
GET /internal/apm/services/{serviceName}/transactions/groups/main_statistics
GET /internal/apm/services/{serviceName}/transactions/groups/detailed_statistics
```

```suggestion
  endpoint: 'GET /internal/apm/services/{serviceName}/errors/groups/main_statistics',
```

Reply:
Actually, nvm. Probably not important.